packaging/osx/datadog-agent

```diff
@@ -89,6 +90,14 @@ case $1 in
         else
             echo "$DESC was not running"
         fi
+        checks=0
+        pid=$(cat $SUPERVISOR_PIDFILE 2>/dev/null)
+        if [ -n $pid ]; then
+            while ps aux $pid | grep $SUPERVISOR_PROCESS >/dev/null && [ $checks -lt 10 ]; do
```

Comment:
(nitpick: we could test `[ $checks -lt 10 ]` first, and then do the `ps aux .....`)